src/server/server.rs

```diff
@@ -33,7 +34,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 128 * REGION_SPLIT_SIZE as usize * 1024 * 1024;
```

Comment:
If you want to use `REGION_SPLIT_SIZE`, use a small multiplier, not `128 * 1024 * 1024`, the bad value here.

Or use a fixed value directly, like 1 GB, 4 GB or 8 GB.